src/styles/cueplayer.css

```diff
@@ -0,0 +1,20 @@
+/* sidepanel tabs */
+.sidepanel > [data-reach-tabs] [data-reach-tab-list] {
```

Comment:
this is totally nitpicking, but I think a `SidePanel` component would be `side-panel` kebab in this context :trollface: